pkgs/by-name/sl/slackdump/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  buildGoModule,
+  lib,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "slackdump";
+  version = "2.5.11";
+
+  src = fetchFromGitHub {
+    owner = "rusq";
+    repo = "slackdump";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-othSIR+KyekUt+/LWhaM5Y6LGsujDCZTfLJotTcPXc0=";
+  };
+
+  vendorHash = "sha256-6GVXzaJ5Lrt4fF0c9cbx6AqpgCwlYgKf6jUJkUyxc0s=";
+
+  meta = {
+    homepage = "https://github.com/rusq/slackdump";
+    changelog = "https://github.com/rusq/slackdump/releases/tag/v${version}";
+    description = "Tools for saving Slack's data without admin privileges";
+    maintainers = with lib.maintainers; [ bot-wxt1221 ];
+    license = lib.licenses.gpl3Plus;
```

Comment:
missing `mainProgram`?

Reply:
you can still have `mainProgram`  when  there are multiple executable, as long as one is clearly considered primary.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

it's possible `$out/bin/slackdump` is a good candidate for `mainProgram` but I'll let @Bot-wxt1221 as maintainer to decide